pyproject.toml

```diff
@@ -129,7 +129,7 @@ pytest-cov = "^2.8.1"
 pytest-localserver = "^0.5.0"
 pytest-sanic = "^1.6.1"
 pytest-asyncio = "^0.10.0"
-pytest-xdist = "^1.31.0"
+pytest-xdist = "^1.32.0"
```

Comment:
This means `1.31.0` is now incompatible?

Reply:
That's a dev dependency, so it doesn't matter — we simply update dev- dependencies to the latest versions if possible.